components/math/algebra/algebrasolvevariable.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useEffect } from "react";
+import { GuessData } from "../../../pages/api/guessData";
+import { Button } from "../../ui/Button";
+
+export interface AlgebraSolveVariableProps {
+  variableLetter: string;
+  variableproblem: string;
+  submitGuess: (guess: GuessData) => void;
+  text: string;
+  answer: string;
+  personname: string;
+}
+
+const AlgebraSolveVariable: React.FC<AlgebraSolveVariableProps> = ({
+  variableLetter,
+  variableproblem,
+  submitGuess,
+  text,
+  answer,
+  personname,
+}) => {
+  const onSubmit = (guess: string) => {
+    submitGuess({
+      guess: guess.toString(),
+      isCorrect: guess === answer,
+    }),
+      setGuess("");
+  };
+  const [guess, setGuess] = useState("");
+  useEffect(() => {
+    (document.getElementById("input") as HTMLInputElement).value = "";
+  }, []);
+  return (
+    <div>
+      <div>
+        <p className="font-size: 30px;">
```

Comment:
Try `text-lg` or `text-xl` here ([link to docs](https://tailwindcss.com/docs/font-size))